client/categories.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Contains the Category Class for NodeBB Client
+"""
+from __future__ import unicode_literals
```

Comment:
kindly read its purpose, i think its not necessary.

Reply:
It's not necessary in our case. Removed.